test_rclcpp/CMakeLists.txt

```diff
@@ -19,8 +19,15 @@ if(AMENT_ENABLE_TESTING)
   set(message_files
     "msg/UInt32.msg"
   )
+
+  set(service_files
+    "srv/AddTwoInts.srv"
+  )
+
   rosidl_generate_interfaces(${PROJECT_NAME}
     ${message_files}
+    ${service_files}
+    DEPENDENCIES builtin_interfaces
```

Comment:
Why is this dependency needed? The new service file seems to only use primitive types.

Same for the package.xml file.